src/react/accordion/Accordion.tsx

```diff
@@ -0,0 +1,75 @@
+'use client'
+
+import clsx from 'clsx'
+import type { JSX, ReactElement } from 'react'
+import { Children, forwardRef } from 'react'
+
+import { mergeRefs } from 'react-merge-refs'
+import { useMODUKComponent } from '../hooks/useMODUKComponent'
+import { AccordionContext } from './AccordionContext'
+import type { AccordionHeadingTag } from './AccordionHeadingTag'
+import type { AccordionItemProps } from './AccordionItem'
+import { AccordionItemIndexContext } from './AccordionItemIndexContext'
+
+export interface AccordionProps {
+  children: ReactElement<AccordionItemProps>[] | ReactElement<AccordionItemProps>
+  headingTag?: AccordionHeadingTag
+  hideAllSectionsText?: string
+  hideSectionText?: string
+  hideSectionAriaLabelText?: string
+  id: string
+  rememberExpanded?: boolean
```

Comment:
Effectively defaults to `true` like the Nunjucks macro (annoying to have a boolean prop that defaults to true though).